### ISSUE TITLE
Add Alacritty to supported terminals

### ISSUE THREAD

**Issue by andradei** [mjswensen/themer#40]
This is the format it uses: https://github.com/alacritty/alacritty/wiki/Color-schemes

Pasted here:

```
# KDE Breeze (Ported from Konsole)
colors:
  # Default colors
  primary:
    background: '#232627'
    foreground: '#fcfcfc'

    dim_foreground: '#eff0f1'
    bright_foreground: '#ffffff'
    dim_background: '#31363b'
    bright_background: '#000000'

  # Normal colors
  normal:
    black: '#232627'
    red: '#ed1515'
    green: '#11d116'
    yellow: '#f67400'
    blue: '#1d99f3'
    magenta: '#9b59b6'
    cyan: '#1abc9c'
    white: '#fcfcfc'

  # Bright colors
  bright:
    black: '#7f8c8d'
    red: '#c0392b'
    green: '#1cdc9a'
    yellow: '#fdbc4b'
    blue: '#3daee9'
    magenta: '#8e44ad'
    cyan: '#16a085'
    white: '#ffffff'

  # Dim colors
  dim:
    black: '#31363b'
    red: '#783228'
    green: '#17a262'
    yellow: '#b65619'
    blue: '#1b668f'
    magenta: '#614a73'
    cyan: '#186c60'
    white: '#63686d'
```

**Comment by mjswensen** [comment]
Working on this now.

**Comment by mjswensen** [comment]
OK, this is available in `themer-alacritty@1.0.0` for CLI, and is deploying to the web UI now.

I think I mapped the colors correctly but please do let me know if you see anything weird (like text with the same color as the background or something 😱)!
